doc/kubernetes/modules/ROOT/pages/running/infinispan-crossdc-deployment.adoc

```diff
@@ -0,0 +1,183 @@
+:ispn: Infinispan
+:ispn-operator: Infinispan Operator
+:kc: Keycloak
+
+{description}
+
+== Audience
+
+This guide describes the procedures required to deploy {ispn} in a multiple cluster environment (cross-site).
+For simplicity, this guide uses the minimum configuration possible that allows {kc} to be used with an external {ispn}.
+
+This guide assumes two {ocp} clusters named `{site-a}` and `{site-b}`
+
+See xref:running/index.adoc[] for additional guides.
+
+== Prerequisites
+
+include::partial$running/infinispan-prerequisites.adoc[]
+
+== Procedure
+
+include::partial$running/infinispan-install-operator.adoc[]
+include::partial$running/infinispan-credentials.adoc[]
++
+Those commands are required both {ocp} clusters.
+Although we use the same credentials in both {ispn} clusters, they could have been different.
+Just adapt your {kc} configuration.
+
+. Create a service account.
++
+A service account is required to establish the cross-site connection.
+The {ispn-operator} uses it to inspect the network configuration from the remote site and to configure the local {ispn} cluster based on it.
++
+For more details check {operator-docs}#managed-cross-site-connections_cross-site[Managing Cross-Site Connections] documentation.
++
+
+.. First, create the service account and generate an access token in both {ocp} clusters.
++
+.Create the service account in `{site-a}`
+[source,bash]
+----
+oc create sa -n ispn-namespace xsite-sa
+oc policy add-role-to-user view -n ispn-namespace -z xsite-sa
+oc create token -n ispn-namespace xsite-sa > site-a-token.txt
+----
++
+.Create the service account in `{site-b}`
+[source,bash]
+----
+oc create sa -n ispn-namespace xsite-sa
+oc policy add-role-to-user view -n ispn-namespace -z xsite-sa
+oc create token -n ispn-namespace xsite-sa > site-b-token.txt
+----
++
+.. The next step is to deploy the token from `{site-a}` into `{site-b}` and vice-versa
++
+.Deploy `{site-b}` token into `{site-a}`
+[source,bash]
+----
+oc create secret generic -n ispn-namespace xsite-token-secret \
+  --from-literal=token="$(cat site-b-token.txt)"
+----
++
+.Deploy `{site-a}` token into `{site-b}`
+[source,bash]
+----
+oc create secret generic -n ispn-namespace xsite-token-secret \
+  --from-literal=token="$(cat site-a-token.txt)"
+----
+
+. Create TLS secrets
++
+In this guide Infinispan uses an {ocp} Route for the cross-site communication.
+It uses the SNI extension of TLS to direct the traffic to the correct Pods.
+To achieve that, JGroups use TLS sockets, which require a Keystore and Truststore with the correct certificates.
++
+For more information check {operator-docs}#securing-cross-site-connections_cross-site[Securing Cross Site Connections] documentation or this https://developers.redhat.com/learn/openshift/cross-site-and-cross-applications-red-hat-openshift-and-red-hat-data-grid[Red Hat Developer Guide].
++
+Upload the Keystore and the Truststore in an {ocp} Secret.
```

Comment:
What do you mean? It is not a header.

Reply:
The statement was so concise, I wasn't sure if it was meant to be a heading for the statements that follow.

Maybe we can expand it slightly so the purpose of this action is clear:

"The Keystore and Truststore must be deployed to {ocp} Secrets so that they can be utilised by the {ispn} deployment."

Then maybe remove the rest of the content before ".Deploy a Keystore".